website/server/docgenHelpers.js

```diff
@@ -1,5 +1,12 @@
 "use strict";
-var b = require('react-docgen/node_modules/recast').types.builders;
+var b;
+try {
+  // npm 3 probably
+  b = require('recast').types.builders;
```

Comment:
Shouldn't this work with npm 2 as well? Specifically, both npm 2 and npm 3 should set up node_modules in ways that are compatible with Node's require algo.